### PR TITLE
added topics for run length encoding

### PR DESCRIPTION
**Pull request by PatrM**
added topics for run length encoding #876 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -432,7 +432,11 @@
       "core": false,
       "difficulty": 5,
       "slug": "run-length-encoding",
-      "topics": null,
+      "topics": [
+        "strings",
+        "integers",
+        "sequences"
```
Comment (FridaTveit) [inline config.json]: Great suggestions :) I agree with `strings` and `integers` but I'm not sure about `sequences`. How do you think that relates to the exercise?

I would also maybe add `transforming` and maybe `pattern_matching`? What do you think? :)

Comment (PatrM) [reply config.json]: good catch, I will add these changes 👍

File: config.json [hunk 2]
```diff
@@ -432,7 +432,12 @@
       "core": false,
       "difficulty": 5,
       "slug": "run-length-encoding",
-      "topics": null,
+      "topics": [
+        "strings",
+        "integers",
+        "pattern_matching",
+	"transforming"
```
Comment (FridaTveit) [inline config.json]: A small indentation issue here. `transforming` should be indented two spaces afer `topics`, same as the other topics :)

**Comment by PatrM** [comment]
sorry, I had some issues with vim and spacing 😅

**Pull request merged**